src/pathfinding_service/cli.py

```diff
@@ -150,6 +176,8 @@ def main(
             registry_address=registry_address,
             sync_start_block=start_block,
             required_confirmations=confirmations,
+            private_key=private_key,
+            poll_interval=10,
```

Comment:
I'd like to not define this here or use a default

Reply:
I defined a `DEFAULT_POLL_INTERVALL = 10` in the pfs config